elements/pf-table/pf-tr.ts

```diff
@@ -110,6 +110,8 @@ export class PfTr extends LitElement {
           <pf-button id="toggle-button"
                      aria-expanded=${String(this.expanded) as 'true' | 'false'}
                      plain
+                     role="button"
+                     aria-label="Expand Button"
```

Comment:
`pf-button` has an API for adding the accessible label to the shadow dom button.  We'd us `label` here instead of using the `aria-label` on the host.
```Suggestion
label="Expand Button"
```